corehq/apps/accounting/async_handlers.py

```diff
@@ -161,14 +161,15 @@ def country_response(self):
         from django_countries.data import COUNTRIES
         countries = sorted(COUNTRIES.items(), key=lambda x: x[1].encode('utf-8'))
         if self.search_string:
-            return filter(lambda x: x[1].lower().startswith(self.search_string.lower()), countries)
+            search_string = self.search_string.capitalize()
```

Comment:
this feels a little more brittle to me than just forcing both to be lowercase, even though it probably runs a tiny bit faster